### ISSUE TITLE
Dashboard. Info and Relationships

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#1453]
## Description of issue

Number of failures. 
On the Dashboard page, the relationship between the number of failures and the corresponding sent notifications (emails, text) is communicated through visual cues such as the position of text, and grouping of content through colour. Without visual information, it may be difficult for non-visual users to know whether “No failures” text refers to the text that precedes or follows. Consider using more explicit text, such as “no email failures”/”no text failures”. 

<img width="1062" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMzJYQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cf6ad9c4b89082c8746f35651524d9ce5cccb862/Capture%20d%E2%80%99e%CC%81cran%2C%20le%202024-01-03%20a%CC%80%2014.16.13.png" alt="Capture d’écran, le 2024-01-03 à 14.16.13.png" />

Describe what is failing
- Relationship relies on visual proximity. 

## Potential fix

- Content could be designed to be more clear and repetitive. 
- We can add visually hidden text

idea: could we design a quick chrome extension to un-hide visually hidden text? Would make it easy to spot problems with them.
